README.md

```diff
@@ -1,18 +1,40 @@
-## FOSSUnited Platform
+<div align="center">
+    <img alt="fossunited logo" src="https://raw.githubusercontent.com/fossunited/Branding/main/asset/general-logos/logo-white.png" width="150px" height="120px">
```

Comment:
The src is not loading up. Maybe try `https://fossunited.org/files/logo618eee.svg`

Reply:
will check the file once again and push out changes.